admin/src/main/resources/io/buoyant/admin/js/src/logger-overrides.js

```diff
@@ -12,6 +12,7 @@ $(function() {
   '        <div id="navbar" class="collapse navbar-collapse">' +
   '          <ul class="nav navbar-nav">' +
   '            <li><a href="../delegator">dtab</a></li>' +
+  '            <li><a href="../requests">requests</a></li>' +
```

Comment:
consider a comment here and/or in `AdminHandler.scala` that the nav's should stay in sync.